### DISCUSSION TITLE
Dropping off a repo (forked, not owned by my account) from the Top Repositories list

### DISCUSSION THREAD

**Discussion by pkmedu** [community/community#75916] [New to GitHub]
Hello, I have a  repository, which I forked (not owned by my account), currently appearing on the GitHub Dashboard page's list of Top Repositories. I want to remove the forked repository from the list.  Upon my inquiry, GitHub support informs me that the forked repository will drop off the list 1 year after I last interacted with it.  Is there a way to know when I interacted with it last?
Thanks,
pkmedu

**Comment by MantasTech** [comment]
Hey pkmedu,

Head to the forked repo on GitHub.

-Click on 'Insights' at the top, then 'Contributors' on the left. You'll see a graph showing your activity.
-Look through 'Pull requests' and 'Issues' tabs for any of your comments or posts.
-Don't forget the 'Commits' tab to see if you've got any recent ones.

That should be it

**Comment by pkmedu** [comment]
Hello,
Thank you for your response.
I looked through Pull Requests and found "There aren't any pull requests."
I looked through Issues, but could not find anything there.
I looked at "Commits" from the side panel but found nothing.
What actions are you suggesting?
regards,
pkmedu


On Mon, Nov 27, 2023 at 12:18 PM Mantas Siukstas ***@***.***>
wrote:

> Hey pkmedu,
>
> In the Repo: Head to the forked repo on GitHub.
> Insights: Click on 'Insights' at the top, then 'Contributors' on the left.
> You'll see a graph showing your activity.
> PRs and Issues: Look through 'Pull requests' and 'Issues' tabs for any of
> your comments or posts.
> Look at Your Commits: Don't forget the 'Commits' tab to see if you've got
> any recent ones.
>
> That should be it
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/community/discussions/75916#discussioncomment-7683540>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AIXDQFSS33LF5VSIXPI52E3YGTDORAVCNFSM6AAAAAA7QXERBGVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TMOBTGU2DA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Pradip K. Muhuri
2824 Big Bear Terrace
Silver Spring, MD 20906
USA
c: 301-728-0290
h: 301-438-0301
email: ***@***.***
